### DISCUSSION TITLE
Restarting Cyb3rVector App development

### DISCUSSION THREAD

**Discussion by phahymn** [cyb3rdog/Cyb3rVector#22] [General]
We see that the project has gone stale. You and randall have done some solid work on this and have almost come to the finish line in one of the phases. (alpha) What happened? Did you get busy? I'm a developer and would like to see how I could help to kickstart this project. Maybe we can get more developers onboard, or get funding. Just wondering what happened and I'm sure we can find a solution for this.

**Comment by moribundant** [comment]
Been stale. Was dealt a KO by the EP 2.0 upgrade. Cyb3rVector was trying to monetize it, and it is a great app (or was). If he opened the source people could get to work on it.
